lua/nvim-lsp-installer/ui/display.lua

```diff
@@ -170,10 +185,12 @@ function M.new_view_only_win(name)
     end
 
     local draw = process.debounced(function(view)
-        -- local win_id = vim.fn.win_findbuf(bufnr)[1]
-        if not win_id or not vim.api.nvim_buf_is_valid(bufnr) then
+        local win_valid = win_id ~= nil and vim.api.nvim_win_is_valid(win_id)
+        local buf_valid = bufnr ~= nil and vim.api.nvim_buf_is_valid(bufnr)
```

Comment:
Checking null here because apparently `vim.api.nvim_win_is_valid(nil) == true` (same for `buf_is_valid`). Apparently it would try to render before the window was opened (I fixed this as well [here](https://github.com/kylo252/nvim-lsp-installer/pull/3/files#diff-be33351e830698e7df89a726a9e09113b1daec1966fde85c18359f1b04df49acR242), so this null check is not currently necessary but better safe than sorry)